src/addons/dragAndDrop/index.js

```diff
@@ -101,5 +102,9 @@ export default function withDragAndDrop(Calendar, {
     endAccessor: accessor
   }
 
-  return DragDropContext(backend)(DragAndDropCalendar);
+  if (withDnDContext) {
```

Comment:
maybe do this when `backend` is `false` (or `null`) and then we don't need to roll in a new option? what do you think?